examples/notebooks/beam-ml/mltransform/scale_data.ipynb

```diff
@@ -0,0 +1,531 @@
+        "\n",
+        "For more information about using `MLTransform`, see [Preprocess data with MLTransform](https://beam.apache.org/documentation/ml/preprocess-data/) in the Apache Beam documentation."
+      ],
+      "metadata": {
+        "id": "F8-yTcjPUHLv"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Import the required modules\n",
+        "\n",
+        "To use `MLTransfrom`, install `tensorflow_transform` and the Apache Beam SDK version 2.53.0 or later."
+      ],
+      "metadata": {
+        "id": "8qhhZTbIyqFb"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "import tempfile\n",
+        "import apache_beam as beam\n",
+        "from apache_beam.ml.transforms.base import MLTransform\n",
+        "from apache_beam.ml.transforms.tft import ScaleTo01\n",
+        "from apache_beam.ml.transforms.tft import ScaleByMinMax\n",
+        "from apache_beam.ml.transforms.tft import ScaleToZScore"
+      ],
+      "metadata": {
+        "id": "y-B7OZ0lyeee"
+      },
+      "execution_count": 3,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "artifact_location_scale_to_01 = tempfile.mkdtemp(prefix='scale_to_01_')\n",
+        "artifact_location_scale_to_zscore = tempfile.mkdtemp(prefix='scale_to_zscore_')\n",
+        "artifact_location_scale_by_min_max = tempfile.mkdtemp(prefix='scale_by_min_max_')"
+      ],
+      "metadata": {
+        "id": "UyBji8pSUEJt"
+      },
+      "execution_count": 4,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "data = [\n",
+        "    {'int_feature_1' : 11, 'int_feature_2': -10},\n",
+        "    {'int_feature_1': 34, 'int_feature_2': -33},\n",
+        "    {'int_feature_1': 5, 'int_feature_2': -63},\n",
+        "    {'int_feature_1': 12, 'int_feature_2': -38},\n",
+        "    {'int_feature_1': 32, 'int_feature_2': -65},\n",
+        "    {'int_feature_1': 63, 'int_feature_2': -21},\n",
+        "]\n",
+        "\n",
+        "\n",
+        "test_data = [\n",
+        "    {'int_feature_1': 29, 'int_feature_2': -20},\n",
+        "    {'int_feature_1': -5, 'int_feature_2': -11},\n",
+        "    {'int_feature_1': 5, 'int_feature_2': -44},\n",
+        "    {'int_feature_1': 29, 'int_feature_2': -12},\n",
+        "    {'int_feature_1': 20, 'int_feature_2': -53},\n",
+        "    {'int_feature_1': 70, 'int_feature_2': -8}\n",
+        "]\n"
+      ],
+      "metadata": {
+        "id": "lNG0WI7Dy1dd"
+      },
+      "execution_count": 5,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Scale the data between 0 and 1\n",
+        "\n",
+        "Scale the data so that it's in the range of 0 and 1. To scale the data, the transform calculates `minimum` and `maximum` values on the whole dataset, and then performs the following calculation:\n",
```

Comment:
```suggestion
        "Scale the data so that it's in the range of 0 and 1. To scale the data, the transform calculates minimum and maximum values on the whole dataset, and then performs the following calculation:\n",
```